### PR TITLE
Fixed lint error

### PR DESCRIPTION
**Pull request by Nityanand13**
#### Summary
Fixed lint error

### CODE REVIEW

File: app/export.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func (a *App) ExportAttachments(attachment *imports.AttachmentImportData, outPat
 
 	if !exists {
 		data := GetMissingAttachmentInfo(attachment)
-		mlog.Warn(fmt.Sprintf("attachment %s not found and will not be exported", *attachment.Path))
+		mlog.Warn("Warn message", mlog.String(fmt.Sprintf("attachment %s not found and will not be exported", *attachment.Path), ""))
```
Comment (Kshitij-Katiyar) [inline app/export.go]: this log message is not similar to the normal log messages in mattermost server

File: app/import.go [hunk 2]
```diff
@@ -52,7 +52,7 @@ func processAttachmentPaths(files []*imports.AttachmentImportData, basePath stri
 			*f.Path = path
 			if len(filesMap) > 0 {
 				if files[i].Data, ok = filesMap[path]; !ok {
-					mlog.Warn(fmt.Sprintf("attachment %s not found and will not be imported", path))
+					mlog.Warn("Warn message", mlog.String(fmt.Sprintf("attachment %s not found and will not be imported", path), ""))
```
Comment (Kshitij-Katiyar) [inline app/import.go]: there should a method name Warnf in the mlog library which can format the log as well see if you can use it

Comment (Nityanand13) [reply app/import.go]: but Warnf is nowhere used in mattermost server

Comment (Nityanand13) [reply app/import.go]: Warnf is not declared by package mlog

Comment (Kshitij-Katiyar) [reply app/import.go]: so what ? if its useful here then use it

**Review by Kshitij-Katiyar — APPROVED** [review]

**Review by manojmalik20 — APPROVED** [review]

**Pull request merged**